plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/driver/DriverLibraryMavenArtifact.java

```diff
@@ -281,7 +281,7 @@ public String getId() {
 
     @Override
     public String getVersion() {
-        if (preferredVersion != null) {
+        if (preferredVersion != null && !preferredVersion.isEmpty()) {
```

Comment:
I think this condition causes an empty version to be shown in various situations.